bears/general/CPDBear.py

```diff
@@ -41,7 +41,7 @@ class CPDBear(GlobalBear):
     def check_prerequisites(cls):  # pragma: no cover
         if which("bash") is None:
             return "bash is not installed."
-        elif which("run.sh") is None:
+        if which("pmd") is None and which("run.sh") is None:
```

Comment:
On the contrary, both calls to `which` will always be called because of the `and`. You want this condition to evaluate to `True` in the case where a user has neither installed, making the Bear unusable.